ddtrace/constants.py

```diff
@@ -34,8 +34,6 @@
 APPSEC_WAF_TIMEOUTS = "_dd.appsec.waf.timeouts"
 APPSEC_WAF_VERSION = "_dd.appsec.waf.version"
 
-APPSEC_RC_ENABLED_ENV = "DD_APPSEC_RC_ENABLED"
```

Comment:
This is a breaking change according to our version policy. Can we keep this constant here and deprecate it instead?

Reply:
Nevermind! This change has not yet been released yet.